docs/enterprise-guide/serialization.md

```diff
@@ -0,0 +1,60 @@
+## Copying contents of one Metabase instance to another
+
+Metabase's serialization feature allows you to create a snapshot, called a dump, of the contents of a Metabase instance that can then be loaded into another instance.
```

Comment:
If someone has a more poetic name than dump, I'm happy to change the (admittedly not very thought-out) terminology.